inductiva/_cli/cmd_storage/remove.py

```diff
@@ -1,20 +1,35 @@
 """Remove the user's remote storage contents via CLI."""
 
+import sys
+
 from inductiva import storage
 
 
 def remove(args):
     """Remove user's remote storage contents."""
-    path = args.path
+    paths = args.path
     confirm = args.confirm
+    all_paths = args.all
+
+    if paths and all_paths:
+        print("inductiva storage remove: error: "
+              "argument path not allowed with argument --all")
+        sys.exit(1)
 
     if not confirm:
-        prompt = input(f"Are you sure you want to remove {path}? (y/n)")
+        if all_paths:
+            prompt = input("Are you sure you want to remove everything "
+                           "from your storage(y/[N])")
```

Comment:
Missing question mark and space after last character:

```suggestion
            prompt = input("Are you sure you want to remove everything "
                           "from your remote storage space (y/[N])? ")
```